mbuild/compound.py

```diff
@@ -35,8 +35,14 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     Parameters
     ----------
     filename : str
-    relative_to_module :
+        Name of the file from which to load atom and bond information. Files
+        are read using the mdtraj package. Please refer to http://mdtraj.org/
```

Comment:
The "Files are read..." sentence probably belongs in the main description not specifically under the filename parameter.